README.md

```diff
@@ -67,17 +67,17 @@ Those are the default settings, that will be overwritten by those set in `settin
 | `ENABLE_REQUEST_HOOK`  | Enable or disable hook on request               | `True` (bool)                                                         |
 | `ENABLE_RESPONSE_HOOK` | Enable or disable hook on request               | `True` (bool)                                                         |
 
-### Available Backends:
+### Available Backends
 * `django_ip_geolocation.backends.IPGeolocationAPI` : (Default) Using https://ipgeolocationapi.com/
 * `django_ip_geolocation.backends.IPStack` : (Require `BACKEND_API_KEY`) Using https://ipstack.com/documentation
 
 
-## Implementing your own backend:
+## Implementing your own backend
 If you want to add a new backend, you need to inherit from `django_ip_geolocation.backends.base`. Then you need to implement `geolocate()` and `_parse()`.
-### `geolocate()`:
-This method will make the external api call. It should also store the api response in `self._raw_data`. and call `self._parse()`.
+### `geolocate()`
+This method will make the external api call. It should also store the api response in `self._raw_data`.
```

Comment:
Better to use present; something like: 'This method [already this words are redundant, because it's clear we discuss about the method which is in title 😄] Makes API calls and stores the API response in...